packages/core/admin/server/policies/hasLicenseLimitAllowance.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const utils = require('@strapi/utils');
+const { ApplicationError } = require('@strapi/utils/lib/errors');
+
+const { PolicyError } = utils.errors;
+
+module.exports = async (policyCtx, config = {}) => {
+  if (!strapi.EE) return true;
+
+  const permittedSeats = strapi.ee.licenseInfo.seats;
+  if (!permittedSeats) return true;
+
+  const userCount = await strapi.db.query('admin::user').count({
```

Comment:
This looks like smth used also in `packages/core/admin/ee/server/controllers/admin.js` :eyes: